kde-apps/filelight/files/filelight-22.04.2-fix_sincos_impl.patch

```diff
@@ -0,0 +1,67 @@
+This patch fixes sincos, runtime issue that causes a SIGSEGV with musl.
```

Comment:
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto

1. Take the commit URL, append .patch, and use _that_ as the patch file (more headers)
2. Include a link to the commit.

Reply:
oh that was useful (1.)